server/liveblog/blogs/blog_membership.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from superdesk.resource import Resource
+from liveblog.common import get_user
+import logging
+from superdesk.activity import add_activity
+from superdesk import get_resource_service
+from flask import g
+from flask import current_app as app, render_template
+from superdesk.emails import send_email
+from liveblog.blogs.blogs import BlogService
+from bson.objectid import ObjectId
+from superdesk.notification import push_notification
+
+logger = logging.getLogger('superdesk')
+
+
+def notify_the_owner(doc, origin):
```

Comment:
I agree with the merge of the two, it doesn't look as bad as I expect, if they are together